README.md

```diff
@@ -43,6 +43,7 @@ compatibility breakages that are outlined below.
 * [Removal of date.timezone Warning](#removal-of-datetimezone-warning)
 * [Removal of Alternative PHP Tags](#removal-of-alternative-php-tags)
 * [Removal of Multiple Default Blocks in Switch Statements](#removal-of-multiple-default-blocks-in-switch-statements)
+* [Removal of Redefinition of parameters with duplicate name](#removal-of-redefinition-of-parameters-with-duplicate-name)
```

Comment:
Just a few minor points :)

Capitalise the title and pluralise "name", to give: "Removal of Redefinition of Parameters with Duplicate Names"